examples/multiwindow.rs

```diff
@@ -14,6 +14,12 @@ fn main() {
         windows.insert(window.id(), window);
     }
 
+    let invisible_window = WindowBuilder::new()
+        .with_visible(false)
+        .build(&event_loop)
+        .unwrap();
+    windows.insert(invisible_window.id(), invisible_window);
+        
```

Comment:
The changes made in this example shouldn't get merged onto `master`.

Reply:
I thought they could be useful to showcase the feature, but I reverted that! YW, the feature will be important for me in one of my projects!

Reply:
TIL, YW is YAAIDK (yet another abbreviation I don't know). :)

Reply:
So is TIL! :)

Reply:
The `cargo fmt` check is failing. Go ahead and rebase against master - we were relying on nightly rustfmt settings beforehand, but we've updated our config to get rid of those flags and only use stable settings. Run `cargo fmt` once this is rebased and Travis should work.

Sorry about the confusion!

Reply:
@chichid (why does your name not get expanded?) TIL is "things / today I learned". So what is "YW"?

Reply:
YW = You're welcome

Reply:
@Osspial No Problem, I will rebase then, once I'm back home! :)